src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -429,7 +429,7 @@ private void checkComment(GHIssueComment comment) throws IOException {
         GHUser sender = comment.getUser();
         String body = comment.getBody();
 
-        logger.log(Level.FINEST, "[{0}] Added comment: {1}", new Object[] { sender.getName(), body });
+        logger.log(Level.FINEST, "[{0}] Added comment: {1}", new Object[] { sender.getName() != null ? sender.getName() : sender.getLogin(), body });
```

Comment:
Minor modification, but could you set a variable so that `sender.getName()` is called once? You could then check the var for null and get the loginID if needed....